js/goto.js

```diff
@@ -1,12 +1,24 @@
+const hasJoinVisited = localStorage.getItem('hasJoinVisited');
+
 function redirectUserToPage(page) {
   const finalPage =
     page == 'signup'
       ? 'https://my.realdevsquad.com/signup?state=get-started'
+      : page == 'join'
+      ? 'https://realdevsquad.com/join'
       : 'https://realdevsquad.com';
 
   window.location.href = finalPage;
 }
 
+function redirectToJoinIfNotVisited() {
```

Comment:
Is this function name a bit confusing? I thought we wanted to go to `join` if we had previously visited it.

Reply:
changed to `redirectHandler()`.